docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,39 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in your meshStack instance. Since this is a BETA
```

Comment:
I dont get the reason why this is beta. It is not planned to have other policies besides the meshObjects

Reply:
Then showing the BETA label in the panel is valid or not?.

Reply:
I think its ok to show the beta label because it is a new thing we have not fully figured out. Just remove the whole remark about the beta.

Reply:
okay.